docs/_articles/ja/soql/soql-builder.md

```diff
@@ -3,7 +3,7 @@ title: SOQL Builder (Beta)
 lang: ja
 ---
 
-SOQL Builder is a VS Code extension that eliminates the guesswork when building SOQL queries. With SOQL Query Builder, anyone can visually build, run, and explore results from queries. Build queries using clicks in a visual editor, then save and extend the queries using a text editor. You can instantly view query results, and then save the results to a `.csv` or `.json` file.
+OQL Builder is a VS Code extension that eliminates the guesswork when building SOQL queries. With SOQL Query Builder, anyone can visually build, run, and explore results from queries. Build queries using clicks in a visual editor, then save and extend the queries using a text editor. You can instantly view query results, and then save the results to a `.csv` or `.json` file.
```

Comment:
Looks like the letter "S" got cutoff. Should be "SOQL" instead of "OQL".

Reply:
I'll take a look and fix. Nice catch! Thanks, @praksb